js/background.js

```diff
@@ -125,22 +130,37 @@ function toBase64 (buffer) {
 }
 
 //
-// Create a catch-all filter so we see all types of content
-//
-var filter = {
-    urls: ["*://*/*"],
-    types: ["main_frame", "sub_frame", "stylesheet", "script", "image", "object", "xmlhttprequest", "other"]
-};
+// Create targetTab record
+// 
+function handleCreated(tab) {
+    targetTab.id = tab.id;
+    targetTab.urls = [];
+    //
+    // Create a catch-all filter so we see all types of content
+    //
+    var filter = {
+        urls: ["*://*/*"],
+        types: ["main_frame", "sub_frame", "stylesheet", "script", "image", "object", "xmlhttprequest", "other"],
+        tabId: tab.id
```

Comment:
I think the `tabId` is also present on the event itself. Maybe if we want to also record events from child/grandchild tabs, we can leave it off here and filter it ourselves inside the listener functions, based on tab hierarchy?